nxdrive/gui/folders_dialog.py

```diff
@@ -402,10 +402,22 @@ def _process_additionnal_local_paths(self, paths: List[str]) -> None:
 
     def _select_more_files(self) -> None:
         """Choose additional local files to upload."""
-        paths, _ = QFileDialog.getOpenFileNames(self, Translator.get("ADD_FILES"))
+        paths, _ = QFileDialog.getOpenFileNames(
+            self,
+            Translator.get("ADD_FILES"),
+            self.engine.dao.get_config("dt_last_local_selected_location", None),
```

Comment:
I mean, pass the value when talking to the `Engine`, that code should not be present at all.